src/Facebook/FacebookRequest.php

```diff
@@ -194,13 +194,13 @@ public function __construct(
 
     $params = ($parameters ?: array());
     if ($session
-      && !isset($params["access_token"])) {
-      $params["access_token"] = $session->getToken();
+      && !isset($params['access_token'])) {
+      $params['access_token'] = $session->getToken();
     }
-    if (FacebookSession::useAppSecretProof()
-      && !isset($params["appsecret_proof"])) {
-      $params["appsecret_proof"] = $this->getAppSecretProof(
-        $params["access_token"]
+    if (!isset($params['appsecret_proof'])
```

Comment:
Space between `!` and `isset` so it's more obvious it's negated. And it's still PSR-2 compliant.